Cargo.toml

```diff
@@ -16,7 +16,7 @@ crc16 = "0.4"
 futures = "0.3"
 pin-project-lite = "0.2"
 rand = { version = "0.8", default-features = false, features = ["std", "std_rng"] }
-redis = { version = "0.21", features = ["tokio-comp"] }
+redis = { version = "0.21", features = ["tokio-comp", "tokio-native-tls-comp"] }
```

Comment:
This forces the tls feature for all users. You should use `cfg)feature = "redis/tokio-native-tls-comp")` to feature gate your changes, (if/where needed) or add an equivalent tls feature to this crate and gate on that instead.

Reply:
Does these changes will help? just to be sure before push

```Cargo.toml
[dependencies]
crc16 = "0.4"
futures = "0.3"
pin-project-lite = "0.2"
rand = { version = "0.8", default-features = false, features = ["std", "std_rng"] }
redis = { version = "0.21", default-features = false }
tokio = { version = "1", features = ["time"] }
log = "0.4"

[dev-dependencies]
anyhow = "1"
once_cell = "1"
tokio = { version = "1", features = ["macros", "full"] }
env_logger = "0.8"
proptest = "0.10"

[features]
default = ["redis/tokio-comp"]
tls = ["redis/tokio-comp","redis/tokio-native-tls-comp"]
```